delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/files/FormDataExtractor.scala

```diff
@@ -41,55 +43,81 @@ object FormDataExtractor {
-    ): IO[FileRejection, (FileDescription, BodyPartEntity)] = {
-      val sizeLimit = Math.min(storageAvailableSpace.getOrElse(Long.MaxValue), maxFileSize)
-      IO.deferFuture(um(entity.withSizeLimit(sizeLimit)))
-        .mapError {
-          case RejectionError(r)                  =>
-            WrappedAkkaRejection(r)
-          case Unmarshaller.NoContentException    =>
-            WrappedAkkaRejection(RequestEntityExpectedRejection)
-          case x: UnsupportedContentTypeException =>
-            WrappedAkkaRejection(UnsupportedRequestContentTypeRejection(x.supported, x.actualContentType))
-          case x: IllegalArgumentException        =>
-            WrappedAkkaRejection(ValidationRejection(Option(x.getMessage).getOrElse(""), Some(x)))
-          case x: ExceptionWithErrorInfo          =>
-            WrappedAkkaRejection(MalformedRequestContentRejection(x.info.format(withDetail = false), x))
-          case x                                  =>
-            WrappedAkkaRejection(MalformedRequestContentRejection(Option(x.getMessage).getOrElse(""), x))
-        }
-        .flatMap { formData =>
-          IO.fromFuture(
-            formData.parts
-              .mapAsync(parallelism = 1) {
-                case part if part.name == fieldName =>
-                  FileDescription(part.filename.getOrElse("file"), part.entity.contentType).runToFuture.map { desc =>
-                    Some(desc -> part.entity)
-                  }
-                case part                           =>
-                  part.entity.discardBytes().future.as(None)
-              }
-              .collect { case Some(values) => values }
-              .toMat(Sink.headOption)(Keep.right)
-              .run()
-          ).mapError {
-            case _: EntityStreamSizeException =>
-              FileTooLarge(maxFileSize, storageAvailableSpace)
-            case th                           =>
-              WrappedAkkaRejection(MalformedRequestContentRejection(th.getMessage, th))
-          }.flatMap(IO.fromOption(_, InvalidMultipartFieldName(id)))
-        }
+  private val defaultContentType: ContentType.Binary = ContentTypes.`application/octet-stream`
+
+  // Creating an unmarshaller defaulting to `application/octet-stream` as a content type
+  implicit private val um: FromEntityUnmarshaller[Multipart.FormData] =
+    MultipartUnmarshallers
+      .multipartUnmarshaller[Multipart.FormData, Multipart.FormData.BodyPart, Multipart.FormData.BodyPart.Strict](
+        mediaRange = `multipart/form-data`,
+        defaultContentType = defaultContentType,
+        createBodyPart = (entity, headers) => Multipart.General.BodyPart(entity, headers).toFormDataBodyPart.get,
+        createStreamed = (_, parts) => Multipart.FormData(parts),
+        createStrictBodyPart =
+          (entity, headers) => Multipart.General.BodyPart.Strict(entity, headers).toFormDataBodyPart.get,
+        createStrict = (_, parts) => Multipart.FormData.Strict(parts)
+      )
```

Comment:
Default from akka is `text/plain`, aligning it to `application/octet-stream` as in the storage part
A bit verbose but it depends on akka there